config/db-config.js

```diff
@@ -0,0 +1,7 @@
+const mongoose = require('mongoose');
+
+module.exports = {
+  DB_NAME: 'wuma-sports',
+  URL: mongoose.connect(`mongodb://localhost/${this.DB_NAME}`, { useNewUrlParser: true }),
```

Comment:
No, just no. Put JUST THE URL here, not the whole connection function. This is a constant.

Reply:
Sorry man... :(